### PR TITLE
Add stekovaya.toml

### PR DESCRIPTION
**Pull request by Stekovaya**

### CODE REVIEW

File: languages/stekovaya.toml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+extensions = [
+  "stk"
+]
+packages = [
+  "mono-complete",
+  "unzip"
+]
+setup = [
+  "cd /usr/src",
+  "wget -q https://repl.it/@theforarkld/Stekovaya.zip",
```
Comment (dan-stowell) [inline languages/stekovaya.toml:12]: You're going to want to version these zip files. I'd hate for you or anyone to edit this repl or the zip file and break the polygott build by accident. Can you pick a good first version number, fork the repl, name it `Stekovaya-some.version.number`, and then link to that repl here?

**Comment by Stekovaya** [comment]
thanks @dan-stowell ! 
i'll did my program, and ill use git command.

File: languages/stekovaya.toml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+entrypoint = "main.stk"
+extensions = [
+  "stk"
+]
+packages = [
+  "mono-complete",
+  "unzip"
+]
+setup = [
+  "cd /usr/src",
```
[the remark this answers is not in the document]
Comment (Stekovaya) [reply languages/stekovaya.toml:11]: wait, i'm using exe file.

File: languages/stekovaya.toml (hunk 1 above)
Comment (Stekovaya) [reply languages/stekovaya.toml:11]: wait sorry, what do you mean?

File: languages/stekovaya.toml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+name = "stekovaya"
+entrypoint = "main.stk"
+extensions = [
+  "stk"
+]
+packages = [
+  "mono-complete",
```
[the remark this answers is not in the document]
Comment (Stekovaya) [reply languages/stekovaya.toml]: i am installing mono-complete for run exe file.

**Pull request closed without merging**